docs/gno-infrastructure/setting-up-a-faucet.md

```diff
@@ -0,0 +1,263 @@
+---
+id: setting-up-a-faucet
+---
+
+# Setting up a faucet for your Gno network
```

Comment:
This tutorial doesn't explain probably the one thing it was supposed to -- how the faucet actually works with subaccounts and funding